Fix the scripts to plot weights

bingyanglin opened the pull request:
- Use weight percentage instead of node counts as y-axis in the DS simulation
- Use percentage as y-axis for confirmation and weight plots
![CT_ZipfParameter_ct](https://user-images.githubusercontent.com/51323441/134516078-847df4ff-5414-4dcf-b93e-953c522aea4c.png)
![CT_ZipfParameter_tp](https://user-images.githubusercontent.com/51323441/134516087-0bfe56a6-46cf-4a7f-bd97-4cbf7501967c.png)
![DS_ZipfParameter_cc](https://user-images.githubusercontent.com/51323441/134516094-561b5008-c82e-43d2-ae5c-25683d1249a8.png)

piotrm50 commented:
`35f338da` this commit deleted a lot of changes that we did with Daria on friday. e.g. we refactored the condition to shutdown the script. Could you take a look at it?

piotrm50 reviewed (changes requested):
One commit with a lot of changes was overwritten. Need to discuss this.

bingyanglin commented:
> `35f338da` this commit deleted a lot of changes that we did with Daria on friday. e.g. we refactored the condition to shutdown the script. Could you take a look at it?

Thanks for pointing this out. I reverted the commit.
